.internal-ci/test/mobilecoind-integration-test.sh

```diff
@@ -25,6 +25,8 @@ pushd "${strategies_dir}" >/dev/null || exit 1
 
 echo "-- Install requirements"
 echo ""
+python3 -m venv venv
+source venv/bin/activate
 pip3 install -r requirements.txt
```

Comment:
Absolutely not advocating this be done now, just planting the seed

We probably want to consider moving over to https://github.com/astral-sh/uv at least for any new python stuff
The main benefits are:
- speed (for local setup) 
- pinning transitive dependencies with a lock file
- One can call out the python version and it will install it
- One can run the tests via one command `uv run pytest` versus this multiline that might leave one in the venv

Reply:
I think if we're going to use something I'd probably go with poetry since we already use it in other projects.